CHANGELOG.md

```diff
@@ -17,6 +11,9 @@
   * Deprecate `BTAppContextSwitcher.sharedInstance.returnURLScheme`
 * BraintreeThreeDSecure
   * Add `BTThreeDSecureRequest.requestorAppURL`
+* BraintreeShopperInsights (BETA)
+  * Add `shopperSessionID` to `BTShopperInsightsClient` initializer
+  * Add `isPayPalAppInstalled()` and/or `isVenmoAppInstalled()`
```

Comment:
I think all of this should be under an `unreleased` section. It did not get released in 6.25.0. Also missing this entry: 
Add `shopperSessionID` to `BTPayPalCheckoutRequest` and `BTPayPalVaultRequest`